crates/matrix-sdk/src/sliding_sync/builder.rs

```diff
@@ -220,15 +222,26 @@ impl SlidingSyncBuilder {
         self
     }
 
+    /// Should the sliding sync instance share its sync position through
+    /// storage?
+    ///
+    /// In general, sliding sync instances will cache the sync position (`pos`
+    /// field in the request) in internal memory. It can be useful, in
+    /// multi-process scenarios, to save it into some shared storage so that one
+    /// sliding sync instance running across two different processes can
+    /// continue with the same sync position it had before being stopped.
+    pub fn share_pos(mut self) -> Self {
```

Comment:
Should we put this method behind `#[cfg(feature = "e2e-encryption")]`? Otherwise it will just have no effect.

Reply:
Agreed for this one...